test/built-ins/Math/prop-desc.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2016 The V8 Project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 20.2
+esid: sec-math-object
+description: >
+  Property descriptor of Math
+info: |
+  The Math Object
+
+  [...]
+  The Math object is not a function object. It does not have a [[Construct]]
+  internal method; it is not possible to use the Math object as a constructor
+  with the new operator. The Math object also does not have a [[Call]] internal
+  method; it is not possible to invoke the Math object as a function.
+
+  17 ECMAScript Standard Built-in Objects:
+
+  Every other data property described in clauses 18 through 26 and in Annex B.2
+  has the attributes { [[Writable]]: true, [[Enumerable]]: false,
+  [[Configurable]]: true } unless otherwise specified.
+includes: [propertyHelper.js]
+features: [Reflect.construct]
+---*/
+
+assert.sameValue(typeof Math, "object", "no [[Call]]");
+
+var obj = Object.defineProperty({}, "length", {
+  get: function() {
+    throw new Test262Error(
+      "Should throw a TypeError if target has no [[Construct]]"
+    );
+  }
+});
+assert.throws(TypeError, function() {
+  Reflect.construct(Math, obj);
```

Comment:
This assertion belongs in a separate file. Also, the details about the second parameter are features of `Reflect.construct`, not `Number`, so they aren't necessary here. I'm not sure that Reflect is necessary, either--have you considered using `new Math`? That would make the test viable for ES5 environments.